packages/python-contract-wrappers/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@0x/python-contract-wrappers",
+    "version": "1.0.0",
```

Comment:
Let's make this a `private` package.

Reply:
https://github.com/0xProject/0x-monorepo/pull/1878/commits/1c8e232d3d8c451d7f2aeadcfaec0763480d4a47